### DISCUSSION TITLE
Welcome to OnLinux Discussions!

### DISCUSSION THREAD

**Discussion by erikaheidi** [erikaheidi/onlinux#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share desk setups and Linux tricks.
  * Engage with other community members.
  * Ask for advice about gadgets and software on Linux.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your desk setup or favorite gadgets.

**Comment by erikaheidi** [comment]
Hi 👋🏼 I'm Erika. I shared my full setup on the website [in this link](https://onlinux.systems/desksetup/erikaheidi) :) Welcome to the group!

**Comment by jonathanbossenger** [comment]
Hi @erikaheidi, congrats on the new site, love this idea. 

I don't do a great job of documenting my setup, mostly I'm from the old PC gamer school of just posting the PC specs on my site ;-)

https://jonathanbossenger.com/current-hardware/

**Reply by jonathanbossenger** [reply]
Forgot to add, I do have a series of posts that details the changes to my work setup over the last 6 years or so, but I've never really created a full "this is my current set up and why" post 

https://jonathanbossenger.com/category/workstation/
